crates/gen/src/type_reader.rs

```diff
@@ -23,6 +23,25 @@ impl TypeReader {
         Self::new(crate::load_winmd::from_os())
     }
 
+    pub fn from_build() -> &'static Self {
+        use std::{mem::MaybeUninit, sync::Once};
+        static ONCE: Once = Once::new();
+        static mut VALUE: MaybeUninit<TypeReader> = MaybeUninit::uninit();
```

Comment:
I think a more common way to do this is with an Option type. This removes the last line of unsafe in favor of a checked unwrap. Slightly less efficient, but also safer. 🤷‍♀️